site/content/entry/yuzu-progress-report-oct-2020/index.md

```diff
@@ -0,0 +1,205 @@
+
+[epicboy](https://github.com/ameerj) changed how `Asynchronous Shaders` [determines the amount of threads to use,](https://github.com/yuzu-emu/yuzu/pull/4865) increasing it for higher thread count CPUs. Past the 8 threads mark, one shader thread is added every two CPU threads.
+
+| CPU Threads | Shader Threads | Example CPUs |
+| :-----------: | :--------------: | ------------ |
+| 1 - 7 | 1 | R3 3200G, i5-9400 |
+| 8 | 2 | R3 3100, i3-10100 |
+| 12 | 4 | R5 3600, i5-10400 |
+| 16 | 6 | R7 3700X, i7-10700K |
+| 32 | 14 | R9 3950X, i9-9960X |
+| 128 | 62 | Threadripper 3990X |
+
+[Kewlan](https://github.com/Kewlan) is back again, fixing another important UI bug. This time, yuzu will [no longer ask for a profile if there is a single one created.](https://github.com/yuzu-emu/yuzu/pull/4817) The devil is in the details.
+
+[Morph](https://github.com/Morph1984) [added a check to see if a directory is a `nullptr`.](https://github.com/yuzu-emu/yuzu/pull/4785) This fixes save file creation in `Hades`. More work is needed to get the game playable.
+
+With the demo just released, [Rodrigo](https://github.com/ReinUsesLisp) started the work to get `Hyrule Warriors: Age of Calamity - Demo Version` playable. By implementing [FCMP immediate](https://github.com/yuzu-emu/yuzu/pull/4853) and [texture CUBE array shadows](https://github.com/yuzu-emu/yuzu/pull/4854) the game starts to render.
+
+{{< imgs
+    "./aoc.png| Where are my glasses? (Hyrule Warriors: Age of Calamity - Demo Version)"
+  >}}
+
+&nbsp;
+
+We personally want to thank [Lioncache.](https://github.com/lioncash) He's not mentioned much in the progress reports due to the technical nature of his work, but he is without a doubt one of the top contributors to the project. Thank you Lion, for the patience you have, and the huge help you give us.
+
+## Future changes
+
+There are some important core fixes incoming, `Local Wireless` emulation continues to progress well, `NVDEC` has more plans in its way, and there is now a roadmap of what will follow the `Texture Cache Rewrite` once it's finished, but we will tell you more about that later.
+
+As an early gift, here's a preview of the Texture Cache Rewrite's progress:
+
+{{< imgs
+    "./tcr.png| "
+  >}}
+
+And that's all folks! Thank you so much for reading, and see you all in the next progress report!
+Thanks RodrigoTR and GG for the pics!
+
+{{< imgs
+    "./burgy.png| "
```

Comment:
I believe the burgy image's creator is https://twitter.com/JamesTurner_42/status/1187581040086441984

Do we want to credit them at all since we're using their asset?

Reply:
I'll do a run with the credit included.

Reply:
Great, definitely think we should provide credit